client/reader/following-manage/connected-subscription-list-item.jsx

```diff
@@ -63,9 +67,12 @@ class ConnectedSubscriptionListItem extends React.Component {
 				url={ url }
 				showEmailSettings={ showEmailSettings && ! isEmailBlocked }
 				showLastUpdatedDate={ showLastUpdatedDate }
+				isFollowing={ isFollowing }
 			/>
 		);
 	}
 }
 
-export default localize( connectSite( ConnectedSubscriptionListItem ) );
+export default connect( ( state, ownProps ) => ( {
```

Comment:
darn, this is where the `connectSite` concept shows its imperfections: still needing to connect it! Still less code duplication than current alternatives 🤷‍♂️  but this wouldn't scale too nicely if we used the pattern everywhere